### PR TITLE
12.0-AFC-2807: Added header button for error reporting.

### PR DESCRIPTION
**Pull request by piratformat**
Please let me know if the sequence needs to be less than 999 on line 14 in file `af_error_reporting/views/assets.xml`.

### CODE REVIEW

File: af_error_reporting/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+{
+    "name": "Af Error Reporting",
+    "summary": "Arbetsförmedlingen error reporting for CRM",
+    "version": "12.0.0.1.0",
+    "category": "Utility",
+    "description": """
```
Comment (draal-af) [inline af_error_reporting/__manifest__.py:9]: Change description to AF uniform,  see example
    'description': """
Jira
===================================================================== \n
V12.0.0.1.0 AFC-2696: Arbetssokande model added \n
    """,

File: af_error_reporting/views/assets.xml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+            <field name="name">Error reporting</field>
+            <field name="type">ir.actions.act_url</field>
+            <field name="target">self</field>
+            <field name="url">https://serviceportal.arbetsformedlingen.se/sp?id=sc_cat_item&amp;sys_id=733449fedb162300c66a547adc9619cf</field>
+        </record>
+
+        <menuitem action="error_reporting_action"
+                  name="Error reporting"
+                  id="menu_error_reporting"
+                  sequence="999"/>
```
Comment (draal-af) [inline af_error_reporting/views/assets.xml:14]: Why the high sequence?

**Review by draal-af — COMMENTED** [review]
Add a link in the Jira to the pull request
Change description in manifest.

Why the high sequence?

File: af_error_reporting/views/assets.xml (hunk 2 above)
Comment (piratformat) [reply af_error_reporting/views/assets.xml:14]: We want this button to allways be last on this list, as far to the right as possible.

**Pull request merged**